web/src/main/java/com/zebrunner/reporting/web/util/LocalDateTimeSerializer.java

```diff
@@ -1,3 +1,18 @@
+/*******************************************************************************
+ * Copyright 2013-2019 Qaprosoft (http://www.qaprosoft.com).
+ *
```

Comment:
Why do we need here copyright, especially Qaprosoft 2019?

Reply:
And in the class above as well

Reply:
No we dont